packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -25,22 +23,15 @@ export const Avatar = (props: AvatarProps) => {
     title,
     initials,
     imageUrl,
-    optimize,
     rounded = true,
-    imageFetchSize = 64,
+    imageFetchSize = 80,
     sx,
     boxElementDescriptor,
     imageElementDescriptor,
   } = props;
   const [error, setError] = React.useState(false);
 
-  let src = imageUrl;
-  if (src && !optimize) {
-    const optimizedHeight = Math.max(imageFetchSize) * (isRetinaDisplay() ? 2 : 1);
-    const srcUrl = new URL(src);
-    srcUrl.searchParams.append('height', optimizedHeight.toString());
-    src = srcUrl.toString();
-  }
+  const src = imageUrl;
```

Comment:
🙃 could we drop this line and use the `imageUrl` instead of `src`?

Reply:
Done!